README.md

```diff
@@ -44,7 +44,7 @@ var options = {
 
 server.register({
   plugin: Good,
```

Comment:
Thanks for catching this. I don't think this change is right either though. I think it should be

```javascript
{
  register: Good,
  options: options
}
```